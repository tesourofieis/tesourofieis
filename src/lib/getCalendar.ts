import { Calendar } from "./src/calendar.ts";

function getCalendar(year: number) {
  const calendar = new Calendar(year);

  return calendar;
}

function getCalendarDay(year: number, date: string) {
  const calendar = new Calendar(year).get(date);

  return calendar;
}

export { getCalendar, getCalendarDay };
