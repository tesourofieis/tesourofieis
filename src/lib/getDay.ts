import { getYear } from "date-fns";
import { Calendar } from "./calendar.ts";
import { format_propers } from "./utils.ts";

function getDay(date: string) {
  const calendar = new Calendar(getYear(date));

  const day = calendar.get(date);

  const proper = day?.getProper(calendar);

  const format = format_propers(proper);

  return format;
}

export { getDay };
