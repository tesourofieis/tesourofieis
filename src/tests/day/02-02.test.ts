import { expect, test } from "vitest";
import { getDay } from "../../lib/getDay";

test("2024-02-02", () => {
  const proper = getDay("2024-02-02");

  expect(proper).toMatchSnapshot();
});
