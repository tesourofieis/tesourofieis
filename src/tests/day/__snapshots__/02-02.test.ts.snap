// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`2024-02-02 1`] = `
[
  {
    "info": {
      "colors": [
        "w",
      ],
      "date": "2024-02-02",
      "day": Day {
        "celebration": [
          Observance {
            "colors": [
              "w",
            ],
            "date": "2024-02-02",
            "flexibility": "sancti",
            "id": "sancti:02-02:2:w",
            "name": "02-02",
            "priority": 46,
            "rank": 2,
            "title": "Purificação da B. V. Maria",
            "weekday": 5,
          },
        ],
        "commemoration": [],
        "date": "2024-02-02",
        "tempora": [
          Observance {
            "colors": [
              "v",
            ],
            "date": "2024-02-02",
            "flexibility": "tempora",
            "id": "tempora:Quadp1-5:4:v",
            "name": "Quadp1-5",
            "priority": 80,
            "rank": 4,
            "title": "Sexta-feira da semana da Septuagésima",
            "weekday": 5,
          },
        ],
      },
      "id": "sancti:02-02:2:w",
      "rank": 2,
      "tempora": "Sexta-feira da semana da Septuagésima",
      "title": "Purificação da B. V. Maria",
    },
    "sections": [
      {
        "body": {
          "latin": [],
          "vernacular": [],
        },
        "id": "Prelude(rubrica 1570 aut rubrica divino)",
        "subSections": [
          {
            "body": {
              "latin": [
                "Finita Tertia, Sacerdos indutus Pluviali violaceo, vel sine Casula, cum Ministris similiter indutis, procedit ad benedicendum Candelas, in medio ante Altare vel ad cornu Epistolæ positas, et ipse, ibidem stans versus ad Altare, dicit manibus junctis in tono Orationis Missæ ferialis:",
                "℣. Dóminus vobíscum.",
                "℟. Et cum spíritu tuo.",
                "Orémus. Domine sancte, Pater omnípotens, ætérne Deus, qui ómnia ex níhilo creásti, et jussu tuo per ópera apum hunc liquorem ad perfectionem cérei veníre fecísti: et qui hodiérna die petitiónem justi Simeónis implésti: te humíliter deprecámur; ut has candélas ad usus hóminum et sanitátem córporum et animárum, sive in terra sive in aquis, per invocatiónem tui sanctíssimi nóminis et per intercessiónem beátæ Maríæ semper Vírginis, cujus hódie festa devóte celebrántur, et per preces ómnium Sanctórum tuórum, bene ☩ dícere et sancti ☩ ficáre dignéris: et hujus plebis tuæ, quæ illas honorífice in mánibus desíderat portare teque cantando laudare, exáudias voces de cœlo sancto tuo et de sede majestátis tuæ: et propítius sis ómnibus clamántibus ad te, quos redemísti pretióso Sánguine Fílii tui:",
                "Qui tecum…",
                "",
                "",
                "Orémus. Omnípotens sempitérne Deus, qui hodiérna die Unigénitum tuum ulnis sancti Simeónis in templo sancto tuo suscipiéndum præsentásti: tuam súpplices deprecámur cleméntiam; ut has candélas, quas nos fámuli tui, in tui nóminis magnificéntiam suscipiéntes, gestáre cúpimus luce accénsas, bene ☩ dícere et sancti ☩ ficáre atque lúmine supérnæ benedictiónis accéndere dignéris: quaténus eas tibi Dómino, Deo nostro, offeréndo digni, et sancto igne dulcíssimæ caritátis tuæ succénsi, in templo sancto glóriæ tuæ repræsentári mereámur.",
                "Per eúndem…",
                "",
                "",
                "Orémus. Dómine Jesu Christe, lux vera, quæ illúminas omnem hóminem veniéntem in hunc mundum: effúnde bene ☩ dictiónem tuam super hos céreos, et sancti ☩ fica eos lúmine grátiæ tuæ, et concéde propítius; ut, sicut hæc luminária igne visíbili accénsa noctúrnas depéllunt ténebras; ita corda nostra invisíbili igne, id est, Sancti Spíritus splendóre illustráta, ómnium vitiórum cæcitáte cáreant: ut, purgáto mentis óculo, ea cérnere possímus, quæ tibi sunt plácita et nostræ salúti utília; quaténus post hujus sǽculi caliginósa discrímina ad lucem indeficiéntem perveníre mereámur. Per te, Christe Jesu, Salvátor mundi, qui in Trinitáte perfécta vivis et regnas Deus, per ómnia sǽcula sæculórum.",
                "℟. Amen.",
                "",
                "",
                "Orémus. Omnípotens sempitérne Deus, qui per Móysen fámulum tuum puríssimum ólei liquórem ad luminária ante conspéctum tuum júgiter concinnánda præparári jussísti: bene ☩ dictiónis tuæ grátiam super hos céreos benígnus infúnde; quaténus sic adminístrent lumen extérius, ut, te donánte, lumen Spíritus tui nostris non desit méntibus intérius.",
                "Per Dominum…",
                "",
                "",
                "Orémus. Dómine Jesu Christe, qui hodiérna die, in nostræ carnis substántia inter hómines appárens, a paréntibus in templo es præsentátus: quem Símeon venerábilis senex, lúmine Spíritus tui irradiátus, agnóvit, suscépit et benedíxit: præsta propítius; ut, ejúsdem Spíritus Sancti grátia illumináti atque edócti, te veráciter agnoscámus et fidéliter diligámus: Qui cum Deo Patre in unitáte ejúsdem Spíritus Sancti vivis et regnas Deus, per ómnia sǽcula sæculórum.",
                "℟. Amen.",
                "",
                "",
                "Finitis Orationibus, Celebrans ponit incensum in thuríbulum: deinde ter aspergit Candelas aqua benedícta, dicendo Antiphonam Aspérges me, sine cantu et sine Psalmo: et ter adolet incenso. Tum dignior ex Clero accedit ad Altare, et ab eo Celebrans accipit Candelam, non genuflectens nec osculans manum illius. Postea Celebrans stans in medio ante Altare versus ad populum, distríbuit Candelas, primum digniori, a quo ipse acceperat; deinde Diacono et Subdiacono paratis, et aliis Clericis singulatim per ordinem, ultimo laicis: omnibus genuflectentibus, Candelam et manum Celebrantis osculantibus, exceptis Prælatis, si adsint. Et cum inceperit distribuere Candelas, a Choro cantatur:",
                "",
                "Antiphona Luc 2:32.",
                "Ant. Lumen ad revelatiónem géntium et glóriam plebis tuæ Israël.",
                "Luc 2:29-31.",
                "Nunc dimíttis servum tuum, Dómine, secúndum verbum tuum in pace",
                "Deinde repetitur tota Antiphona “Lumen ad revelatiónem géntium” quæ similiter repetitur post quemlibet Versum.",
                "Ant. Lumen ad revelatiónem géntium et glóriam plebis tuæ Israël.",
                "Quia vidérunt óculi mei salutáre tuum.",
                "Ant. Lumen ad revelatiónem géntium et glóriam plebis tuæ Israël.",
                "Quod parásti ante fáciem ómnium populorum.",
                "Ant. Lumen ad revelatiónem géntium et glóriam plebis tuæ Israël.",
                "Glória Patri, et Fílio, et Spirítui Sancto.",
                "Ant. Lumen ad revelatiónem géntium et glóriam plebis tuæ Israël.",
                "Sicut erat in princípio, et nunc, et semper, et in sǽcula sæculórum. Amen.",
                "Ant. Lumen ad revelatiónem géntium et glóriam plebis tuæ Israël.",
                "",
                "",
                "His expletis, cantatur:",
                "Antiph Ps 43:26.",
                "Ant. Exsúrge, Dómine, ádjuva nos: et líbera nos propter nomen tuum.",
                "Ps 43:2.",
                "Deus, áuribus nostris audívimus: patres nostri annuntiavérunt nobis.",
                "Glória Patri…",
                "Ant. Exsúrge, Dómine, ádjuva nos: et líbera nos propter nomen tuum.",
                "",
                "",
                "Deinde Sacerdos dicit:",
                "Orémus.",
                "Et si fuerit post Septuagesimam, et non in die Dominica, Diaconus dicit:",
                "℣. Flectámus génua.",
                "℟. Leváte.",
                "Exáudi, quǽsumus, Dómine, plebem tuam: et, quæ extrinsécus ánnua tríbuis devotióne venerári, intérius asséqui grátiæ tuæ luce concéde. Per Christum, Dóminum nostrum.",
                "℟. Amen.",
                "",
                "",
                "Deinde fit Processio. Et primo Celebrans ponit incensum in thuribulum: postea Diaconus vertens se ad populum, dicit:",
                "℣. Procedámus in pace.",
                "℟. In nómine Christi. Amen.",
                "Præcedit Thuriferarius cum thuribulo fumiganti: deinde Subdiaconus paratus, deferens Crucem, medius inter duos Acolythos cum candelabris accensis: sequitur Clerus per ordinem, ultimo Celebrans cum Diacono a sinistris, omnes cum Candelis accensis in manibus: et cantantur Antiphonæ sequentes:",
                "Ant. Adórna thálamum tuum, Sion, et súscipe Regem Christum: ampléctere Maríam, quæ est cœléstis porta: ipsa enim portat Regem glóriæ novi lúminis: subsístit Virgo, addúcens mánibus Fílium ante lucíferum génitum: quem accípiens Símeon in ulnas suas, prædicávit pópulis, Dóminum eum esse vitæ et mortis et Salvatórem mundi.",
                "",
                "Luc 2:26, 27 et 28-29.",
                "Ant. Respónsum accépit Símeon a Spíritu Sancto, non visúrum se mortem, nisi vidéret Christum Dómini: et cum indúcerent Púerum in templum, accépit eum in ulnas suas, et benedíxit Deum, et dixit: Nunc dimíttis servum tuum, Dómine, in pace.",
                "℣. Cum indúcerent púerum Jesum parentes ejus, ut fácerent secúndum consuetúdinem legis pro eo, ipse accépit eum in ulnas suas.",
                "Et ingrediendo Ecclesiam, cantatur:",
                "℣. Obtulérunt pro eo Dómino par túrturum, aut duos pullos columbárum: * Sicut scriptum est in lege Dómini.",
                "℣. Postquam impléti sunt dies purgatiónis Maríæ, secúndum legem Moysi, tulérunt Jesum in Jerúsalem, ut sísterent eum Sicut scriptum est in lege Dómini.",
                "℣. Glória Patri, et Fílio, et Spirítui Sancto.",
                "℟. Sicut scriptum est in lege Dómini.",
                "Finita Processione, Celebrans et Ministri, depositis violaceis, accipiunt paramenta alba pro Missa. Et Candelæ tenentur in manibus accensæ, dum legitur Evangelium, et iterum ab incepto Canone usque ad expletam Communionem.",
                "Hodie prohibetur quælibet Missa votiva, etiam sollemnis, de Christo Domino.",
                "",
              ],
              "vernacular": [
                "Se a bênção das velas e a procissão não poderem ser feitas na sua forma solene, devido à ausência de ministros sagrados, é aceitável fazer-se da forma simples, até sem canto, desde que pelo menos três acólitos assistam o celebrante. O celebrante paramenta-se com capa branca ou sem casula, e segue com os ministros que se vestem similarmente para abençoar as velas, que são colocadas no centro do altar ou no lado da epístola. Em pé, de frente para o altar, o padre canta sem entoação, com as mão unidas:",
                "℣. O Senhor seja convosco.",
                "℟. E com o vosso espírito.",
                "Oremos. Senhor santo, Pai omnipotente, Deus eterno, que criastes todas as coisas do nada e por ordem de quem as abelhas compõem a substância para se formarem estas velas, e que neste dia atendestes à petição do justo Simeão: humildemente Vos rogamos, pela invocação do vosso santo nome, pela intercessão da B. Maria, sempre Virgem, cuja festividade hoje celebramos piedosamente, e pelas preces de todos vossos Santos, que Vos digneis benzer ☩ e santificar ☩ estas velas para uso dos homens e para a saúde dos corpos e das almas, quer na terra, quer no mar. Ouvi, lá do vosso celestial santuário e do trono da vossa majestade, os votos do vosso povo, aqui presente, que deseja levar reverentemente em suas mãos estas velas e louvar-Vos com seus cânticos; enfim, sede propício para com todos que por Vós clamam, os quais resgatastes pelo precioso Sangue do vosso Filho:",
                "Que convosco vive...",
                "",
                "",
                "Oremos. Deus omnipotente e eterno, que hoje apresentastes o vosso Filho Unigénito no vosso templo para que fosse recebido nos braços de S. Simeão, incessantemente suplicamos à vossa clemência se digne benzer ☩, santificar ☩ e acender com a luz da vossa bênção estas velas que nós, vossos servos, desejamos levar acesas, depois de as havermos recebido para honra do vosso santo Nome, a fim de que, oferecendo-as a Vós, que sois nosso Deus e Senhor, nos tornemos dignos e sejamos abrasados no fogo sagrado da vossa suavíssima caridade, e depois mereçamos ser apresentados no templo sagrado da vossa glória.",
                "Pelo mesmo nosso Senhor...",
                "",
                "",
                "Oremos. Senhor Jesus Cristo, luz verdadeira que ilumina todo o homem que vem a este mundo, lançai a vossa bênção ☩ sobre estas velas e santificai-as ☩ com a luz da vossa graça; permiti benigno que, assim como estes círios, brilhando com luz visível, afastam as trevas da noite, assim também os nossos corações, iluminados com o fogo invisível, isto é, ilustrados com o esplendor do Espírito Santo sejam livres da cegueira de todos os vícios, a fim de que, purificados os olhos da nossa alma, possamos conhecer o que Vos é agradável e útil à nossa salvação; e, assim, após as perigosas trevas deste mundo, mereçamos alcançar a posse da luz eterna. Por Vós, ó Jesus Cristo, Salvador do mundo, que, sendo Deus, na Trindade perfeita, viveis e reinais por todos os séculos dos séculos.",
                "℟.: Amen.",
                "",
                "",
                "Oremos. Deus omnipotente e eterno, que mandastes preparar por Moisés, vosso servo, óleo puríssimo para as lâmpadas, que incessantemente deviam arder na Vasa presença, infundi benigno a graça da vossa bênção ☩ nestas velas, de sorte que, dando-nos elas a luz exterior, não falte interiormente às nossas almas pela vossa graça a luz do vosso Espírito Santo.",
                "Por nosso Senhor...",
                "",
                "",
                "Oremos. Senhor Jesus Cristo, que, aparecendo hoje entre os homens na substância da nossa carne, fostes por vossos Pais apresentado no templo, e a quem o venerável Simeão, ilustrado pela luz do vosso Espírito, aceitou em suas mãos, reconheceu e abençoou, concedei-nos propício que, iluminados e ilustrados pela graça do Espírito Santo, verdadeiramente Vos reconheçamos e fielmente Vos amemos.",
                "℟.: Amen.",
                "",
                "",
                "O celebrante asperge as velas três vezes com água benta, dizendo a antífona Asperges me, e incenssado-as também três vezes.",
                "",
                "Antífona Lc 2:32",
                "Ant. A luz que iluminará as nações e a glória de Israel, vosso povo.",
                "Lc 2:29-31",
                "Agora, Senhor, deixai ir em paz o vosso servo, segundo a vossa palavra.",
                "Repete-se a Ant. "A luz que iluminará..."",
                "Ant. A luz que iluminará as nações e a glória de Israel, vosso povo.",
                "Pois os meus olhos já viram a vossa salvação.",
                "Ant. A luz que iluminará as nações e a glória de Israel, vosso povo.",
                "Que preparastes diante dos olhos de todos os povos.",
                "Ant. A luz que iluminará as nações e a glória de Israel, vosso povo.",
                "Glória ao Pai, e ao Filho e ao Espírito Santo.",
                "Ant. A luz que iluminará as nações e a glória de Israel, vosso povo.",
                "Assim como era no princípio, agora e sempre, e por todos os séculos dos séculos. Amen.",
                "Ant. A luz que iluminará as nações e a glória de Israel, vosso povo.",
                "",
                "",
                "Depois canta-se:",
                "Ant. Sl 43:26.",
                "Ant. Erguei-Vos, Senhor, auxiliai-nos e salvai-nos, pela honra do vosso nome.",
                "Sl 43:26",
                "Ó Deus, ouvimos com nossos ouvidos, nossos pais contaram-nos as vossas maravilhas.",
                "Glória ao Pai.",
                "Ant. Erguei-Vos, Senhor, auxiliai-nos e salvai-nos, pela honra do vosso nome.",
                "",
                "",
                "Depois o padre diz:",
                "Oremos.",
                "Se o dia 2 de Fevereiro for depois da Septuagésima, e não ao Domingo, diz-se:",
                "℣. Ajoelhemos!",
                "℟. Levantai-vos!",
                "Dignai-vos ouvir o vosso povo, Senhor, e fazei que pela luz da vossa graça realizemos nas nossas almas o mystério que nos permitis celebrar com esta homenagem anual da nossa piedade. Por Cristo, nosso Senhor.",
                "℟. Amen.",
                "",
                "",
                "Se existir procissão, primeiro, o celebrante coloca incenso turíbulo, depois o diácono vira-se para o povo e diz:",
                "℣. Caminhemos em paz.",
                "℟. Em nome de Cristo. Amen.",
                "O incensador vai primeiro, carregando o turíbulo, seguido pelo subdiácono, que carrega a cruz entre dois acólitos com velas acesas, depois os clérigos por ordem, finalmente o celebrante com o diácono à sua esquerda. Todos seguram velas acesas. A seguinte antífona é cantada:",
                "Ant. Prepara a tua alcova, ó Sião, e recebe Cristo-Rei. Acolhe com amor Maria, que é a porta do céu; pois leva consigo o Rei da glória, o Rei da luz nova. Maria permanece Virgem e apresenta o seu Filho, que foi gerado antes da aurora. Simeão recebe-os nos seus braços e anuncia ao povo que está ali o Senhor da vida e da morte - o Salvador do mundo.",
                "",
                "Lc 2:26, 27 et 28-29.",
                "Ant. Simeão havia sido avisado pelo Espírito Santo de que não morreria antes de ver o Ungido do Senhor. E, quando levaram o Menino ao templo, recebeu-o em seus braços, louvou a Deus e disse:  «Agora, Senhor, deixai ir em paz o vosso servo».",
                "℣. Levando seus Pais o menino Jesus ao templo para observarem a seu respeito os costumes da lei, Simeão recebeu-O em seus braços.",
                "Quando a procissão reentra na Igreja, canta-se:",
                "℣. Ofereceram por Ele ao Senhor um par de rolas ou dois pombinhos * Como está escrito na lei do Senhor.",
                "℣. Cumpridos os dias da purificação de Maria, segundo a lei de Moisés, levaram Jesus a Jerusalém, para O apresentarem ao Senhor: * Como está escrito na lei do Senhor.",
                "℣. Glória ao Pai, e ao Filho e ao Espírito Santo.",
                "℟. Como está escrito na lei do Senhor.",
                "Quando se completa a procissão, o celebrante e ministros colocam os paramentos para a Missa. Na missa que se segue à bênção das velas as orações ao pé do altar são omitidas, assim como "Aufer novis" e "Oramus te, Domine". Quando o padre vai para o altar, imediamente sobe e beija-o no centro. Seguram-se as velas durante o Evangelho e durante o Canon.",
                "",
              ],
            },
            "id": "Benedictio Candelarum",
          },
        ],
      },
      {
        "body": {
          "latin": [
            "Ps 47:10-11.",
            "Suscépimus, Deus, misericórdiam tuam in médio templi tui: secúndum nomen tuum, Deus, ita et laus tua in fines terræ: justítia plena est déxtera tua.",
            "Ps 47:2.",
            "Magnus Dóminus, et laudábilis nimis: in civitáte Dei nostri, in monte sancto ejus.",
            "Glória Patri…",
            "Suscépimus, Deus, misericórdiam tuam in médio templi tui: secúndum nomen tuum, Deus, ita et laus tua in fines terræ: justítia plena est déxtera tua.",
            "",
          ],
          "vernacular": [
            "Sl 47:10-11",
            "Recebemos, ó Deus, a vossa misericórdia no meio do vosso templo. Assim como o vosso nome, assim os vossos louvores, ó Deus, ressoam até às extremidades da terra: a vossa dextra está cheia de justiça.",
            "Sl 47:2",
            "O Senhor é grande e digno de todos os louvores, na cidade do nosso Deus, na sua montanha sagrada.",
            "Glória ao Pai.",
            "Recebemos, ó Deus, a vossa misericórdia no meio do vosso templo. Assim como o vosso nome, assim os vossos louvores, ó Deus, ressoam até às extremidades da terra: a vossa dextra está cheia de justiça.",
            "",
          ],
        },
        "id": "Introitus",
        "subSections": [],
      },
      {
        "body": {
          "latin": [
            "Omnípotens sempitérne Deus, majestátem tuam súpplices exorámus: ut, sicut unigénitus Fílius tuus hodiérna die cum nostræ carnis substántia in templo est præsentátus; ita nos fácias purificátis tibi méntibus præsentári.",
            "Per eúndem…",
            "",
          ],
          "vernacular": [
            "Deus omnipotente e eterno, humildemente suplicamos à vossa majestade que, assim como o vosso Filho Unigénito foi neste dia apresentado no templo na substância da nossa carne, assim também sejamos apresentados diante de Vós com nossas almas limpas.",
            "Por nosso Senhor...",
            "",
          ],
        },
        "id": "Oratio",
        "subSections": [],
      },
      {
        "body": {
          "latin": [
            "Léctio Malachíæ Prophétæ.",
            "Malach 3:1-4.",
            "Hæc dicit Dóminus Deus: Ecce, ego mitto Angelum meum, et præparábit viam ante fáciem meam. Et statim véniet ad templum suum Dominátor, quem vos quæritis, et Angelus testaménti, quem vos vultis. Ecce, venit, dicit Dóminus exercítuum: et quis póterit cogitáre diem advéntus ejus, et quis stabit ad vidéndum eum? Ipse enim quasi ignis conflans et quasi herba fullónum: et sedébit conflans et emúndans argéntum, et purgábit fílios Levi et colábit eos quasi aurum et quasi argéntum: et erunt Dómino offeréntes sacrifícia in justítia. Et placébit Dómino sacrifícium Juda et Jerúsalem, sicut dies sǽculi et sicut anni antíqui: dicit Dóminus omnípotens.",
            "",
          ],
          "vernacular": [
            "Lição do Profeta Malaquias.",
            "Ml 3:1-4",
            "O Senhor Deus disse: «Eis que envio o meu Anjo, que preparará o caminho diante da minha face. E logo virá ao seu templo o Dominador, que procurais, e o Anjo da Aliança, que desejais. Eis que Ele vem, diz o Senhor dos Exércitos. Mas quem poderá adivinhar o dia da sua vinda? Quem estará lá para O ver? Pois Ele será como o fogo, que funde os metais, ou como a erva, de que se servem os lavandeiros. Assentar-se-á como um homem que funde e limpa a prata; purificará os filhos de Levi e os tornará limpos, como o ouro ou a prata, e oferecerão ao Senhor sacrifícios de justiça. E o sacrifício de Judá e de Jerusalém será agradável ao Senhor, como nos séculos passados e nos anos antigos: isto diz o omnipotente Senhor».",
            "",
          ],
        },
        "id": "Lectio",
        "subSections": [],
      },
      {
        "body": {
          "latin": [
            "Ps 47:10-11; 47:9.",
            "Suscépimus, Deus, misericórdiam tuam in médio templi tui: secúndum nomen tuum, Deus, ita et laus tua in fines terræ.",
            "℣. Sicut audívimus, ita et vídimus in civitáte Dei nostri, in monte sancto ejus.",
            "Luc 2:29-32.",
            "Nunc dimíttis servum tuum, Dómine, secúndum verbum tuum in pace.",
            "℣. Quia vidérunt óculi mei salutáre tuum.",
            "℣.. Quod parásti ante fáciem ómnium populórum.",
            "℣.. Lumen ad revelatiónem géntium et glóriam plebis tuæ Israël.",
            "",
          ],
          "vernacular": [
            "Sl 47:10-11, 9.",
            "Recebemos, ó Deus, a vossa misericórdia no meio do vosso templo. Assim como o vosso nome, ó Deus, assim os vossos louvores ressoam até às extremidades da terra.",
            "℣. Aquilo que havia sido anunciado, vimo-lo na cidade do nosso Deus, na sua montanha sagrada.",
            "Lc 2:29-32",
            "Agora deixareis, Senhor, ir em paz o vosso servo.",
            "℣. Pois os meus olhos já viram a vossa salvação:",
            "℣. Que preparastes diante dos olhos de todos os povos.",
            "℣. A luz que iluminará as nações e a glória de Israel, vosso povo.",
            "",
          ],
        },
        "id": "Tractus",
        "subSections": [],
      },
      {
        "body": {
          "latin": [
            "Sequéntia ☩ sancti Evangélii secúndum Lucam.",
            "Luc 2:22-32.",
            "In illo témpore: Postquam impleti sunt dies purgatiónis Maríæ, secúndum legem Moysi, tulérunt Jesum in Jerúsalem, ut sísterent eum Dómino, sicut scriptum est in lege Dómini: Quia omne masculínum adapériens vulvam sanctum Dómino vocábitur. Et ut darent hóstiam, secúndum quod dictum est in lege Dómini, par túrturum aut duos pullos columbárum. Et ecce, homo erat in Jerúsalem, cui nomen Símeon, et homo iste justus et timorátus, exspéctans consolatiónem Israël, et Spíritus Sanctus erat in eo. Et respónsum accéperat a Spíritu Sancto, non visúrum se mortem, nisi prius vidéret Christum Dómini. Et venit in spíritu in templum. Et cum indúcerent púerum Jesum parentes ejus, ut fácerent secúndum consuetúdinem legis pro eo: et ipse accépit eum in ulnas suas, et benedíxit Deum, et dixit: Nunc dimíttis servum tuum, Dómine, secúndum verbum tuum in pace: Quia vidérunt óculi mei salutáre tuum: Quod parásti ante fáciem ómnium populórum: Lumen ad revelatiónem géntium et glóriam plebis tuæ Israël.",
            "",
          ],
          "vernacular": [
            "Continuação ☩ do santo Evangelho segundo S. Lucas.",
            "Lc 2:22-32",
            "Naquele tempo, quando acabaram os dias da purificação de Maria, levaram Jesus a Jerusalém, segundo a lei de Moisés, para O apresentar ao Senhor, como na lei do Senhor está escrito: «Todo o masculino, quando nascer, será consagrado ao Senhor», e para darem a oferta, segundo o que está escrito na lei do Senhor: «Um par de rolas ou dois pombinhos». Ora havia em Jerusalém um homem, chamado Simeão, que era justo, temente a Deus e esperava a consolação de Israel; e no qual habitava o Espírito Santo, que lhe inspirara que não haveria de morrer sem que visse o Ungido do Senhor. Veio, então, ao templo, conduzido pelo Espírito; e, como os Pais conduzissem o Menino Jesus para que n’Ele se cumprisse o que a lei ordenava, tomou Simeão o Menino nos braços e louvou a Deus, dizendo: «Agora, Senhor, deixai ir em paz o vosso servo, segundo a vossa palavra, pois os meus olhos já viram o vosso Salvador, que preparastes ante a face de todos os povos: a luz para iluminar as nações e a glória de Israel, vosso povo».",
            "",
          ],
        },
        "id": "Evangelium",
        "subSections": [],
      },
      {
        "body": {
          "latin": [
            "Ps 44:3.",
            "Diffúsa est grátia in lábiis tuis: proptérea benedíxit te Deus in ætérnum, et in sǽculum sǽculi.",
            "",
          ],
          "vernacular": [
            "Sl 44:3",
            "A graça espalhou-se nos vossos lábios; por isso abençoou-vos Deus para sempre.",
            "",
          ],
        },
        "id": "Offertorium",
        "subSections": [],
      },
      {
        "body": {
          "latin": [
            "Exáudi, Dómine, preces nostras: et, ut digna sint múnera, quæ óculis tuæ majestátis offérimus, subsídium nobis tuæ pietátis impénde.",
            "Per Dominum…",
            "",
          ],
          "vernacular": [
            "Ouvi nossas preces, Senhor; e, a fim de que as ofertas que apresentamos diante dos olhos da vossa majestade, sejam dignas, concedei-nos o auxílio da vossa misericórdia.",
            "Por nosso Senhor...",
            "",
          ],
        },
        "id": "Secreta",
        "subSections": [],
      },
      {
        "body": {
          "latin": [
            "de Nativitate Domini",
            "Vere dignum et justum est, æquum et salutáre, nos tibi semper et ubíque grátias ágere: Dómine sancte, Pater omnípotens, ætérne Deus: Quia per incarnáti Verbi mystérium nova mentis nostræ óculis lux tuæ claritátis infúlsit: ut, dum visibíliter Deum cognóscimus, per hunc in invisibílium amorem rapiámur. Et ídeo cum Angelis et Archángelis, cum Thronis et Dominatiónibus cumque omni milítia cæléstis exércitus hymnum glóriæ tuæ cánimus, sine fine dicéntes.",
            "",
          ],
          "vernacular": [
            "do Natal",
            "Verdadeiramente é digno e justo, racional e salutar dar-Vos graças, Senhor santo, Pai omnipotente, eterno Deus, em todos os lugares e sempre, porque pelo mystério do Verbo Incarnado um novo clarão do vosso esplendor iluminou a nossa alma, a fim de que, ao passo que conhecemos Deus de uma maneira visível, sejamos inspirados por Ele no amor às cousas invisíveis. E, por isso, com os Anjos e Arcanjos, com os Tronos e Dominações e com toda a milícia do exército celestial, cantamos o hino da vossa glória, dizendo incessantemente:",
            "",
          ],
        },
        "id": "Prefatio",
        "subSections": [],
      },
      {
        "body": {
          "latin": [
            "Luc 2:26.",
            "Respónsum accépit Símeon a Spíritu Sancto, non visúrum se mortem, nisi vidéret Christum Dómini.",
            "",
          ],
          "vernacular": [
            "Lc 2:26",
            "Simeão recebera do Espírito Santo a revelação de que não morreria sem ver o Ungido do Senhor.",
            "",
          ],
        },
        "id": "Communio",
        "subSections": [],
      },
      {
        "body": {
          "latin": [
            "Quǽsumus, Dómine, Deus noster: ut sacrosáncta mystéria, quæ pro reparatiónis nostræ munímine contulísti, intercedénte beáta María semper Vírgine, et præsens nobis remédium esse fácias et futúrum.",
            "Per Dominum…",
            "",
          ],
          "vernacular": [
            "Ó Senhor, nosso Deus, Vos suplicamos, permiti por intercessão da B. Maria, sempre Virgem, que os sacrossantos mystérios que nos concedestes, como salvaguarda da nossa regeneração, nos sirvam de remédio para o presente e para o futuro.",
            "Por nosso Senhor...",
            "",
          ],
        },
        "id": "Postcommunio",
        "subSections": [],
      },
    ],
  },
]
`;
